### ISSUE TITLE
Add notification support for Mattermost

### ISSUE THREAD

**Issue by alitari** [keel-hq/keel#143]
First of all, thanks for this great tool! In our company we use [Mattermost ](https://about.mattermost.com/), so we would be very happy if we could receive nofications  from keel there.

As far as i see, the implementation could be quite simple:

Mattermost  wants a different payload structure of the webhook. Instead of 
```
{
	"name":"
	"message"
	"createdAt"
	"type"
	"level"
}
```
it needs: 
```
{
         "text":
}
```
See also https://docs.mattermost.com/developer/webhooks-incoming.html

**Comment by rusenask** [comment]
Hi, thanks! :) Their API seems very nice, love the markdown idea. I will get the server somewhere and try it out. 

Do you care about notifications or approvals (might be a bit more work for approvals) as well?

**Comment by alitari** [comment]
First of all i'm interested in simple notifications.

**Comment by rusenask** [comment]
Hey, it seems that mattermost supports Slack webhooks https://docs.mattermost.com/developer/webhooks-incoming.html :
> Mattermost incoming webhooks are Slack-compatible. You can copy-and-paste code used for a Slack incoming webhook to create Mattermost integrations. Mattermost automatically translates the Slack’s proprietary JSON payload format.

Could you try it out?

**Comment by alitari** [comment]
Hmm, maybe i have a wrong understanding of the keel configuration, but for slack configuration i can't configure an endpoint, only a token (which i don't have). So, I can't imagine how this can work...

**Comment by rusenask** [comment]
Ahh sorry, my bad. My memory is not as good as it used to be :D

**Comment by rusenask** [comment]
Available in `0.6.0`, docs: https://keel.sh/v1/guide/documentation.html#Mattermost-notifications
